TASVideos/WikiModules/BannedUsers.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@model BannedUsers
+
+<standard-table condition="User.Has(PermissionTo.AssignRoles)">
+	@*
```

Comment:
The query needs to run SortedPageOf() instead of ToList(), then this sortable table head should "just work"

Reply:
It did just work for pagination but not sorting.